### PR TITLE
Travis-CI test fix

### PR DESCRIPTION
**Pull request by nekrich**
- README.md update

- swiftlint config added

- Removed unused DZNEmptyDataSet dependency

- Reduced compile time

- removed Podfile.lock

- Removed XCGLogger pod integration leftovers

### CODE REVIEW

File: Hyber/Hyber.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1122,59 +1213,40 @@
...
 			defaultConfigurationName = Release;
 		};
-		809A13D01C6CA7A900CB3977 /* Build configuration list for PBXNativeTarget "HyberTests" */ = {
+		809A13CD1C6CA7A900CB3977 /* Build configuration list for PBXNativeTarget "Hyber" */ = {
 			isa = XCConfigurationList;
 			buildConfigurations = (
-				809A13D11C6CA7A900CB3977 /* Debug */,
-				809A13D21C6CA7A900CB3977 /* Release */,
+				809A13CE1C6CA7A900CB3977 /* Debug */,
+				809A13CF1C6CA7A900CB3977 /* Release */,
```
Comment (arodiss) [inline Hyber/Hyber.xcodeproj/project.pbxproj]: @nekrich 
This file seems to be auto-generated
If this is the case, it should not be in VCS

File: Source/Core/Extensions/NSURLRequest+cURLPresentation.swift [hunk 2]
```diff
@@ -37,53 +37,54 @@ internal extension NSURLRequest {
...
   {
-    var components = ["$ curl -i"]
+    var components: [String] = ["$ curl -i"]
     
     guard let
       //        request = self.request,
-      URL = URL,
-      host = URL.host
+      URL = URL//,
+//      host = URL.host
```
Comment (arodiss) [inline Source/Core/Extensions/NSURLRequest+cURLPresentation.swift]: @nekrich 
commented code is generally bad thing in repo - if you don't need something, just remove it

File: Source/Core/Extensions/NSURLRequest+cURLPresentation.swift [hunk 3]
```diff
@@ -37,53 +37,54 @@ internal extension NSURLRequest {
...
+//        realm: host,
+//        authenticationMethod: NSURLAuthenticationMethodHTTPBasic
+//      )
+//
+//      if let credentials = credentialStorage.credentialsForProtectionSpace(protectionSpace)?.values {
+//        credentials.forEach {
+//          components.append(String(format: "-u %@:%@", $0.user!, $0.password!))
+//        }
+//      }
+//    }
```
Comment (arodiss) [inline Source/Core/Extensions/NSURLRequest+cURLPresentation.swift]: @nekrich 
more commented code

File: Source/Core/Extensions/NSURLRequest+cURLPresentation.swift [hunk 4]
```diff
@@ -37,53 +37,54 @@ internal extension NSURLRequest {
...
+//    if configuration.HTTPShouldSetCookies {
+//      if let
+//        cookieStorage = configuration.HTTPCookieStorage,
+//        cookies = cookieStorage.cookiesForURL(URL)
+//        where !cookies.isEmpty
+//      {
+//        let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value ?? String());" }
+//        components.append("-b \"\(string.substringToIndex(string.endIndex.predecessor()))\"")
+//      }
+//    }
```
Comment (arodiss) [inline Source/Core/Extensions/NSURLRequest+cURLPresentation.swift]: @nekrich 
more commented code

**Comment by arodiss** [comment]
@nekrich 
I suggest you to delete branches after merge

File: Hyber/Hyber.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nekrich) [inline Hyber/Hyber.xcodeproj/project.pbxproj]: This stings are links to build Configurations generated by CocoaPods, i can't manage them. I renamed CocoaPods target, so we have new string.

**Pull request merged**